.github/workflows/bok-choy-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: Bok Choy Tests
+
+on:
+  push:
+    branches:
+     - master
+  pull_request:
+
+jobs:
+  run_tests:
+    name: Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Set display to virtual frame buffer
+      run: export DISPLAY=:99.0
+
+    - name: Set up test environment
+      run: |
+        sudo apt-get update
+        sudo apt-get install xvfb gettext
+        wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
+        mkdir geckodriver
+        tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
+        export PATH=$PATH:$PWD/geckodriver
+        export BOKCHOY_HEADLESS=true
+
+    - name: Install dependencies
+      run: |
+        pip install -r requirements/base.txt
```

Comment:
do we've to install python dependencies to run bok-choy tests?

Reply:
Yes we need the dependencies to run browser tests. Otherwise it raises Module not found errors